blueprint/index.md

```diff
@@ -1 +1,210 @@
+   3. Specify the GUID of the Genesys Cloud open messaging integration in the `integrations` array. Replace the existing value with the GUID that is already present for `chat` and `offers` objects.
+   4. Provide the `userId` in the format of the email address of the user. 
+For more information about installing and running the project using the Ionic toolkit, see [here](https://github.com/shansrini/ionic-mobile-open-messaging-blueprint/blob/main/ionic-frontend/README.md).
+
+If you have set with Ionic and `npm`, you can navigate to the ionic-frontend folder and execute `ionic serve` in the command prompt.
+
+### How it works?
+
+Once you have set up everything, send a message from the chat widget of the Ionic front-end interface. The chat message is routed to Genesys Cloud Inbound Message flow. The agent responds from the Genesys Cloud agent desktop and Genesys Cloud sends the message to the webhook URL defined in the open messaging integration.
+
+**Agentless setup**
+If you want to send an agentless message, you must invoke the Genesys Cloud API. For more information, see [Conversations](https://developer.genesys.cloud/api/rest/v2/conversations/#post-api-v2-conversations-messages-agentless "Opens the Conversations page") in Genesys Cloud Developer Center.
+
+**Offers**
+You can use the **Offers** tab in the Ionic front-end interface to push any information or coupons to your customers. Replies are not allowed for such messages. To use the **Offers** option, post a message to your Amazon API Gateway using the Amazon API gateway endpoint that you have created.
+```
+POST <Amazon API Gateway URL>/notification/integration/{integrationId}/user/{userId}
+```
+From the Ionic configuration file, get the following details:
+* `integrationId` - The ID from the integrations section for the `Offers` type.
+* `userId` - The email address of the user.
+
+Use the following format for the POST request body text:
+```
+{
+    "type": "text",
+    "content": "SuperCoupon",
+    "fromId": "agentId"
+}
+```
+**QR code support**
+The application can render QR codes in both the **Connect** and the **Offers** tab of the Ionic front-end interface. Use the following format from the Genesys Cloud Agent desktop or the Agentless API in the body text:
+```
+    "gcDecoratedMessage": {
+        "gcMessageType": "qrCode",
+        "gcMessageText": "SuperSpecialCoupon!"
+    }
+}
+```
+In the **Offers** tab, you can POST this message body to the Amazon API Gateway URL `<Amazon API Gateway URL>/notification/integration/{integrationId}/user/{userId}`:
+```
+{
+    "type": "qrcode",
+    "content": "SuperCoupon",
+    "fromId": "agentId"
+}
+```
+## Additional resources
+* [Amazon API Gateway](https://aws.amazon.com/api-gateway/ "Opens the Amazon API Gateway page") in the Amazon featured services
+* [AWS Lambda](https://aws.amazon.com/translate/ "Opens the Amazon AWS Lambda page") in the Amazon featured services
```

Comment:
This link takes you to the AWS Translate product. I think you intended to use: https://aws.amazon.com/lambda/

Reply:
fixed